### PR TITLE
Derived icon changes and reset sorting fix

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4049

### Change description ###
Changing derived icon for release-2
Adding in derived icon for all work

ALSO - fixed reset sorting issue when persisting task list

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
LGTMc

File: src/work-allocation-2/containers/task-list/task-list.component.ts [hunk 1]
```diff
@@ -184,8 +186,14 @@ export class TaskListComponent implements OnChanges {
...
   public onResetSorting(): void {
+    let header = document.getElementById(`header_${this.sortedBy.fieldName}`);
+    header.setAttribute('aria-sort', 'none');
+    this.pagination.page_number = 1;
     this.setDefaultSort();
     this.sortEvent.emit(this.sortedBy.fieldName);
+    this.paginationEvent.emit(this.pagination.page_number);
+    header = document.getElementById(`header_${this.sortedBy.fieldName}`);
+    header.setAttribute('aria-sort', 'ascending');
   }
```
Comment (phillip-whitaker) [inline src/work-allocation-2/containers/task-list/task-list.component.ts:197]: This method is interesting... Why do you set the aria-sort attribute and then immediately take them off?

**Review by phillip-whitaker — APPROVED** [review]
LGBTQ+

File: src/work-allocation-2/containers/task-list/task-list.component.ts (hunk 1 above)
Comment (connorpgpmcelroy) [reply src/work-allocation-2/containers/task-list/task-list.component.ts:202]: Have added comments

**Pull request merged**